tests/test_config.py

```diff
@@ -199,6 +199,16 @@ def test_loading_keys_from_environment_variables(self):
         config.ready = True
         assert config.get("num_participants") == 1
 
+    def test_loading_auto_recruit_from_redis(self):
+        config = Configuration()
```

Comment:
I think instead of instantiating the config manually like this it would be best practice to use the pre-existing `active_config` fixture.

Reply:
Done